hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -8,14 +8,290 @@ title: "OTP Controller Technical Specification"
+  - Some partitions can be locked statically (by computing and storing an associated digest in OTP), others can be read locked at runtime via a CSRs.
+- Write lockability
+  - This controls whether a partition is locked and prevented from future updates.
+  - A locked partition is stored alongside a digest to be used later for integrity verification.
+- Integrity Verification
+  - Once a partition is write-locked via its digest, it can undergo periodic verification (time-scale configurable by software).
+This verification takes two forms, partition integrity checks, and storage consistency checks.
+
+Since the OTP is memory-like in nature (it only outputs a certain number of bits per address location), some of the logical partitions are buffered in registers for instantaneous and parallel access by hardware.
+This is a critical point, since after power-up, these particular OTP contents are stored in flip flops and sourced to the system.
+I.e., buffered partitions are **NOT** directly sourced from the OTP macro itself.
+Thus the security of both volatile (OTP controller) and non-volatile (OTP IP) storage becomes important.
+
+### Partition Listing and Description
+
+The OTP controller for OpenTitan contains the seven logical partitions shown below.
+
+Partititon     | Secret | Buffered | WR Lockable  | RD Lockable  | Description
+---------------|--------|----------|--------------|--------------|----------------------------------------------------------------
+CREATOR_SW_CFG | no     | no       | yes (digest) |   yes (CSR)  | Software configuration partition for device-specific calibration data (Clock, LDO, RNG, device identity).
+OWNER_SW_CFG   | no     | no       | yes (digest) |   yes (CSR)  | Software configuration partition for data that changes software behavior, specifically in the ROM. <br> E.g., enabling defensive features in ROM or selecting failure modes if verification fails.
+HW_CFG         | no     | yes      | yes (digest) |      no      | Hardware configuration bits used to hardwire specific hardware functionality. <br> E.g., raw entropy accessibility or flash scrambling bypass range.
+LIFE_CYCLE     | no     | yes      |     no       |      no      | Life-cycle related bits (**TODO ADD link**). **Note**, this partition cannot be locked as the life-cycle state needs to be able to advance to RMA in-field.
+SECRET0        | yes    | yes      | yes (digest) | yes (digest) | Test unlock tokens.
+SECRET1        | yes    | yes      | yes (digest) | yes (digest) | SRAM and FLASH scrambling keys.
+SECRET2        | yes    | yes      | yes (digest) | yes (digest) | RMA unlock token and creator root key.
+
+Generally speaking, the production life cycle of a device is split into 5 stages "Manufacturing" -> "Calibration and Testing" -> "Provisioning" -> "Mission" -> "RMA".
+OTP values are usually programmed during "Calibration and Testing", "Provisioning" and "RMA" stages, as explained below.
```

Comment:
looks like "Calibration and Testing" and "Provisioning" have sessions below to talk to OTP behavior, do you think it is necessary to have a session for "RMA" as well?